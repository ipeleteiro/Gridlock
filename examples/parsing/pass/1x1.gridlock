GRIDLOCK
Players: Amy, Bill.
Grid: 1x1.
Colours: red, green.
Amy plays red at (0,0).
Bill cannot move.
Amy wins!