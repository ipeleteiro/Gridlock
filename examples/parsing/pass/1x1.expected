GameRecord {players = ["Amy","Bill"], grids = [Grid {width = 1, height = 1, rep = fromList [((0,0),Empty)]},Grid {width = 1, height = 1, rep = fromList [((0,0),Filled Red)]}], colours = fromList [Red,Green]}